lib/validic.rb

```diff
@@ -16,5 +16,13 @@ def configure
       yield self
       true
     end
+
+    def api_url
+      @api_url ||= "https://api.validic.com"
```

Comment:
Super nit pick, but not really important because there is no linting or CI set up for this yet, but in general ruby (and the most popular linter rubocop's default settings) prefers single quotes unless doing string interpolation.